### DISCUSSION TITLE
Save Revision as New Item

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#7934] [Feature Requests]
You can revert an item back to a previous state, but it would be nice to have the option to save a previous revision state as a _new_ item. It's like "Save as Copy", but for revisions. I am imagining this as an action button in the revision item drawer header.

**Comment by br41nslug** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
